src/EditorFeatures/Core/SolutionEvents/HostLegacySolutionEventsWorkspaceEventListener.cs

```diff
@@ -11,29 +11,29 @@
 using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
 using Microsoft.CodeAnalysis.Host;
 using Microsoft.CodeAnalysis.Host.Mef;
-using Microsoft.CodeAnalysis.Options;
 using Microsoft.CodeAnalysis.Remote;
 using Microsoft.CodeAnalysis.Shared.TestHooks;
-using Microsoft.CodeAnalysis.SolutionCrawler;
 using Roslyn.Utilities;
 
 namespace Microsoft.CodeAnalysis.LegacySolutionEvents;
 
+/// <summary>
+/// Event listener that hears about workspaces and exists solely to let unit testing continue to work using their own
```

Comment:
> exists solely to let unit testing continue to work

Is it possible to move this type to test project?

Reply:
Sorry. By "unit testing" I don't mean "our unit tests".  I mean "the unit testing team". They have a crawler to discover tests.  But it's extremely trimmed down, and owned by them.